common/rya.api/src/main/java/org/apache/rya/api/instance/RyaDetailsToConfiguration.java

```diff
@@ -53,14 +53,16 @@ public static void addRyaDetailsToConfiguration(final RyaDetails details, final
         checkAndSet(conf, ConfigurationFields.USE_FREETEXT, details.getFreeTextIndexDetails().isEnabled());
       //RYA-215        checkAndSet(conf, ConfigurationFields.USE_GEO, details.getGeoIndexDetails().isEnabled());
         checkAndSet(conf, ConfigurationFields.USE_TEMPORAL, details.getTemporalIndexDetails().isEnabled());
-        PCJIndexDetails pcjDetails = details.getPCJIndexDetails();
-		if (pcjDetails.isEnabled() && pcjDetails.getFluoDetails().isPresent()) {
-			checkAndSet(conf, ConfigurationFields.USE_PCJ_UPDATER, true);
-			conf.set(ConfigurationFields.FLUO_APP_NAME, pcjDetails.getFluoDetails().get().getUpdateAppName());
-			conf.set(ConfigurationFields.PCJ_UPDATER_TYPE, "FLUO");
-		} else {
-			checkAndSet(conf, ConfigurationFields.USE_PCJ_UPDATER, false);
-		}
+        final PCJIndexDetails pcjDetails = details.getPCJIndexDetails();
+        if (pcjDetails.isEnabled() && pcjDetails.getFluoDetails().isPresent()) {
+            checkAndSet(conf, ConfigurationFields.USE_PCJ_UPDATER, true);
+            conf.set(ConfigurationFields.FLUO_APP_NAME, pcjDetails.getFluoDetails().get().getUpdateAppName());
+            conf.set(ConfigurationFields.PCJ_UPDATER_TYPE, "FLUO");
+            conf.set(ConfigurationFields.PCJ_STORAGE_TYPE, "ACCUMULO");
+        } else {
+            checkAndSet(conf, ConfigurationFields.USE_PCJ_UPDATER, false);
+            conf.set(ConfigurationFields.PCJ_UPDATER_TYPE, "NO_UPDATE");
```

Comment:
I realize it was already like this but make use of the PrecomputedJoinUpdaterType and PrecomputedJoinStorageType enums in this method instead of hard-coded strings.
Replace:
"NO_UPDATE" with PrecomputedJoinUpdaterType.NO_UPDATE.toString(),
"FLUO" with PrecomputedJoinUpdaterType.FLUO.toString(), and
"ACCUMULO" with PrecomputedJoinStorageType.ACCUMULO.toString()